uptane/common.py

```diff
@@ -27,6 +44,17 @@ def sign_signable(signable, keys_to_sign_with):
     keys_to_sign_with:
       A list whose elements must conform to tuf.formats.ANYKEY_SCHEMA.
 
+    datatype:
```

Comment:
Ah, the next block (displayed immediately below this in the review) specifies 'json' and 'der', but this one doesn't. Sorry. Fixing.